modules/utxo-lib/src/noble_crypto.ts

```diff
@@ -0,0 +1,80 @@
+import * as createHash from 'create-hash';
+import * as necc from '@noble/secp256k1';
+import { MuSigFactory, MuSig } from '@brandonblack/musig';
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+// @ts-ignore base_crypto is exported as a subPath export, ignoring since compiler complains about importing like this
+import * as base_crypto from '@brandonblack/musig/base_crypto';
+
+const crypto = {
+  ...base_crypto,
+  pointMultiplyUnsafe: (p: Uint8Array, a: Uint8Array, compress: boolean): Uint8Array | null => {
+    try {
+      const product = necc.Point.fromHex(p).multiplyAndAddUnsafe(
+        necc.Point.ZERO,
+        BigInt(`0x${Buffer.from(a).toString('hex')}`),
```

Comment:
this is a bit hacky - let's add a ``toBigInt(b: Uint8Array | Buffer): BigInt` instead that uses https://nodejs.org/api/buffer.html#bufreadbiguint64beoffset (or LE)